using_openshift/templates.adoc

```diff
@@ -49,20 +49,23 @@ A template describes a set of resources intended to be used together that can be
 <4> A list of labels to apply to resources
 <5> A list of resources to create
 
+====
+
 == Labels
-The list of labels specified in the template will be applied to every resource that is generated from the template. link:../architecture/kubernetes_model.html#label[Labels] can be used to manage and organize generated resources.
+link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template will be applied to every resource that is generated from the template.
 
 == Parameters
-The list of parameters that you want to override are listed in the *parameters* section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+The list of parameters that you can override are listed in the parameters section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
 
 ****
-`osc process --parameters -f [replaceable]#<filename>#`
+`$ osc process --parameters -f _<filename>_`
 ****
 
-The following shows the output when listing the parameters for one of the [replaceable]#openshift/origin/examples# template.
+The following shows the output when listing the parameters for one of the `_openshift/origin/examples_` templates.
```

Comment:
I see that `openshift/origin/examples` was marked up as a replaceable in a much earlier commit. So your markup change here would be correct in that case, however I think this was originally meant to refer to the `examples` directory in the `openshift/origin` repo, ie https://github.com/openshift/origin/tree/master/examples. You can see in the example output that follows that the command is processing the `examples/sample-app/application-template-dockerbuild.json` file.

So I think  `openshift/origin/examples` could either be a direct link to the above URL, or just be marked up as a directory instead (bold+italic).